### ISSUE TITLE
user request: support for links between other diagrams in VS Code

### ISSUE THREAD

**Issue by pleabargain** [mjbvz/vscode-markdown-mermaid#130]
I have two diagrams in my dir
test.md
proj.md

I want to be able to link between one diagram and another inside VS Code.
thank you
Dennis
```
```mermaid
graph LR;
    A-->proj2;
    click proj2 "C:\Users\denni\OneDrive\Documents\mermaid_play\test.md"
``` 
```

**Comment by mjbvz** [comment]
Is this supported in mermaid?

**Comment by pleabargain** [comment]
Click to a different URL is supported in Mermaid.

**Comment by mjbvz** [comment]
What about relative links? Supporting absolute file paths is out of scope as they would not work in html  and are not portable

**Comment by stefanfritsch** [comment]
Yes, according to [this stackoverflow answer](https://stackoverflow.com/questions/60195408/is-there-a-way-to-put-urls-in-the-same-document-using-mermaid-markdown-flowchart).

Relative file links would be brilliant. Mermaid would be so useful as tool to document a repo structure.

**Comment by o-o00o-o** [comment]
I have exactly the same requirement. It works find if I publish to Azure Devops wiki, but not in vscode preview right now forcing me to have a bunch of disconnected links to the md's I want to link to. It would be an infinitely better user experience to be able to click on the diagram nodes

**Comment by Cinerar** [comment]
With current development of AI codding agents, this would be brilliant to analyze code that were created, by asking agent to create mermaid diagram with links to exact implementations.
